cirq-core/cirq/contrib/acquaintance/executor.py

```diff
@@ -100,6 +106,47 @@ def optimization_at(
         )
 
 
+@cirq.transformer
+def strategy_executor(
+    circuit: 'cirq.Circuit',
+    *,
+    execution_strategy: ExecutionStrategy = None,
+    context: Optional['cirq.TransformerContext'] = None,
+) -> None:
+    """Executes an acquaintance strategy"""
```

Comment:
Extend the docstring to document the args (`circuit`, `context`, `execution_strategy`) and types of errors it raises.